x-pack/plugins/apm/public/components/app/TransactionDetails/Transaction/TransactionPropertiesTableForFlyout.tsx

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+// @ts-ignore
+import { EuiTab } from '@elastic/eui';
+import { capitalize, first, get } from 'lodash';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { SERVICE_AGENT_NAME } from '../../../../../common/constants';
+import { Transaction } from '../../../../../typings/Transaction';
+import { IUrlParams } from '../../../../store/urlParams';
+// @ts-ignore
+import { fromQuery, history, toQuery } from '../../../../utils/url';
+import {
+  getPropertyTabNames,
+  PropertiesTable
+} from '../../../shared/PropertiesTable';
+// @ts-ignore
+import { Tab } from '../../../shared/UIComponents';
+
+// Ensure the selected tab exists or use the first
+function getCurrentTab(tabs: string[] = [], selectedTab: string | null) {
+  return selectedTab && tabs.includes(selectedTab) ? selectedTab : first(tabs);
+}
+
+function getTabs(transactionData: Transaction) {
+  const dynamicProps = Object.keys(transactionData.context);
+  return getPropertyTabNames(dynamicProps);
+}
+
+interface Props {
+  location: Location;
+  transaction: Transaction;
+  urlParams: IUrlParams;
+}
+
+export const TransactionPropertiesTableForFlyout: React.SFC<Props> = ({
```

Comment:
What does `React.SFC` give us, that we can't get with a function?

Reply:
It marks the function as a React stateless functional component which allows for adding things like propTypes, defaultValues, and displayName.